src/main/java/easton/Action.java

```diff
@@ -0,0 +1,8 @@
+package easton;
+
+/**
+ * Represents that the chatbot can handle.
```

Comment:
I think the JavaDoc could be more descriptive to better explain what `Action` represents. A possible revision could be 
`*Represents the different types of actions/commands that the chatbot can handle.`